### DISCUSSION TITLE
Which files to edit for doc PR?

### DISCUSSION THREAD

**Discussion by Jmuccigr** [ImageMagick/ImageMagick#4108] [Help] [answered]
If one wanted to do a PR for the documentation, which is the file to work on? Is it Website/include/command-line-options.php?

**Comment by dlemstra** [comment, marked as answer]
Our websites can be found here: https://github.com/ImageMagick/Website and here: https://github.com/ImageMagick/Website6

**Reply by Jmuccigr** [reply]
Thanks. I got that, but I meant **specifically** which files, because sometimes docs are generated from one set and so it does no good to try to edit the final versions. Is there a particular dir with the files to edit?

**Reply by dlemstra** [reply]
It depends on what you want to change but the Website project only contains files that are used to create the docs and our website so feel free to edit any file in that website repository.
